tensorflow/examples/image_retraining/retrain.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-r"""Simple transfer learning with Inception v3 or Mobilenet models.
+"""Simple transfer learning with Inception v3 or Mobilenet models.
```

Comment:
Isn't the r intentional to make it a raw string?

Reply:
I think so… 
Sorry, I close the pull request.